pycuda/driver.py

```diff
@@ -207,6 +212,9 @@ def _build_arg_buf(args):
             elif isinstance(arg, np.void):
                 arg_data.append(_my_bytes(_memoryview(arg)))
                 format += "%ds" % arg.itemsize
+            elif hasattr(arg, "__cuda_array_interface__"):
+                arg_data.append(arg.__cuda_array_interface__["data"][0])
```

Comment:
Would it be more "pythonic" to just `getattr(..., None)` or `try: getattr(...)` below? What do you think?

Reply:
It probably would be - will revise accordingly.

Reply:
I've switched to using `getattr(..., None)` below.